src/runtime/config/configuration-qemu-snp.toml.in

```diff
@@ -643,7 +676,7 @@ experimental=@DEFAULTEXPFEATURES@
 #
 # Offload the CRI image management service to the Kata agent.
 # (default: false)
-service_offload = @DEFSERVICEOFFLOAD@
+#service_offload = true
```

Comment:
We don't want this change from main. We disable image service offload in main but it should work in CCv0.